tiles/src/main/java/com/protomaps/basemap/layers/Landuse.java

```diff
@@ -89,7 +134,7 @@ public void processFeature(SourceFeature sf, FeatureCollector features) {
 
   @Override
   public List<VectorTile.Feature> postProcess(int zoom, List<VectorTile.Feature> items) throws GeometryException {
-    items = Area.addAreaTag(items);
+    //items = Area.addAreaTag(items);
```

Comment:
¿Do I need this for the later filter to work?

Reply:
You don't  - the reason it exists was at some point style-side filtering was happening based on area. If Tilezen doesn't need something like this, we can remove it.